uxarray/grid.py

```diff
@@ -160,33 +160,57 @@ def __from_ds__(self, dataset):
             raise RuntimeError("unknown file format: " + self.mesh_filetype)
         dataset.close()
 
-    def write(self, outfile, grid_type):
+    def write(self, outfile, grid_type, save_as='netcdf'):
         """Writes mesh file as per extension supplied in the outfile string.
 
         Parameters
         ----------
         outfile : str, required
-            Path to output file
+            Name of or path to output file. If path is not given,
+            file is saved to current working directory
+
+            ex: "path/to/new/file/location/new_file.nc" or
+                "new_file.nc"
+
         grid_type : str, required
             Grid type of output file.
             Currently supported options are "ugrid", "exodus", and "scrip"
 
+        save_as : str, default "netcdf"
+            The specific file type to save newly created datasets to.
+            Current options are "netcdf", "zarr", and "None" (no file saved)
```

Comment:
`zarr` package needs to be added into dependencies (also shouldn't be forgotten to be added in the conda-forge recipe). It is the reason of `to_zarr` call not working correctly.